book/onchain-verification/contract-addresses.md

```diff
@@ -16,9 +16,28 @@ To verify SP1 proofs on-chain, we recommend using our deployed verifier gateways
 | 534351   | Scroll Sepolia   | [0x3B6041173B80E77f038f3F2C0f9744f04837185e](https://sepolia.scrollscan.com/address/0x3B6041173B80E77f038f3F2C0f9744f04837185e)         |
 | 534352   | Scroll           | [0x3B6041173B80E77f038f3F2C0f9744f04837185e](https://scrollscan.com/address/0x3B6041173B80E77f038f3F2C0f9744f04837185e)                 |
 
-A complete reference for all of the SP1Verifier contract addresses can be also be found in the [SP1 Contracts Repo](https://github.com/succinctlabs/sp1-contracts/blob/main/contracts/deployments).
+A complete reference for all of the `SP1Verifier` contract addresses can be also be found in the [SP1 Contracts Repo](https://github.com/succinctlabs/sp1-contracts/blob/main/contracts/deployments).
```

Comment:
nit: We don't need to re-link here. IMO, we should have one source of truth (the SP1 docs) that holds all of this. We should remove this link.

Reply:
If we want to have one source of truth, then we should just have the sp1-contracts `deployments` dir and remove this table from the docs. The docs will *always* lag behind the deployments json.

Reply:
though it seems most reasonable to have both and just link to the deployments as a backup in the worst case that a user needs more information (e.g. a specific SP1Verifier address)

Reply:
Sure, then you can say: "The most up-to-date reference for the specific versions supported on each chain can be found in the [deployments] on the SP1 contracts repository. On mainnets, only official versioned releases are supported. 

We should always strive to keep the public-facing docs up to date. We can just make it a part of the deployment process as something that must be added. E.g. When you add a new chain, make a PR to add it to the SP1 docs.

Reply:
ok sg